src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor().toString();
```

Comment:
```suggestion
        String color = colorSupplier.getRandomColor().name();
```
from common mistakes

Reply:
accidentally requested, sorry